docs/conf.py

```diff
@@ -48,6 +48,9 @@
 
 autoapi_dirs = ['../notfound']
 autoapi_add_toctree_entry = False
+def setup(app):
```

Comment:
This function would be good to be moved to the bottom of the file just to follow the convention.

Reply:
Thanks for the review. I moved this section and added a comment/header.